layers/+intl/japanese/packages.el

```diff
@@ -0,0 +1,93 @@
+;;; packages.el --- Japanese Layer packages File for Spacemacs
+;;
+;; Copyright (c) 2012-2016 Sylvain Benner & Contributors
+;;
+;; Author: Sylvain Benner <sylvain.benner@gmail.com>
+;; URL: https://github.com/syl20bnr/spacemacs
+;;
+;; This file is not part of GNU Emacs.
+;;
+;;; License: GPLv3
+
+;; List of all packages to install and/or initialize. Built-in packages
+;; which require an initialization must be listed explicitly in the list.
+(setq japanese-packages
+      '(helm
```

Comment:
Here you state that your layer uses `helm`, but there is no `japanese/[post|pre]-?init-helm` function. So this is redundant.